src/mumble/Settings.cpp

```diff
@@ -323,7 +323,7 @@ Settings::Settings() {
 	bOnlyAttenuateSameOutput       = false;
 	bAttenuateLoopbacks            = false;
 	iMinLoudness                   = 1000;
-	iVoiceHold                     = 50;
+	iVoiceHold                     = 20;			// Mic hold time = iVoiceHold / 100 (seconds)
```

Comment:
```suggestion
	iVoiceHold                     = 20;			// Actual mic hold time is (iVoiceHold / 100) seconds
```
Note that this comment should probably also mention that `iVoiceHold` is specified in "frames" instead of in a concrete unit of time. E.g. something like `specified in terms of "frames" each of which has a size of iFrameSize.`